cpu/cc2538/ldscripts/cc2538sf53.ld

```diff
@@ -15,13 +15,18 @@
  *
  * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
+ 
+_rom_offset = DEFINED( _rom_offset ) ? _rom_offset : 0x0;
+_fw_rom_length = DEFINED( _fw_rom_length ) ? _fw_rom_length : _rom_length - _rom_offset;
+
+ASSERT((_fw_rom_length <= _rom_length - _rom_offset), "Specified firmware size does not fit in ROM");
```

Comment:
This has now been put in common with `cortexm_rom_offset.ld` so you can reuse it

https://github.com/RIOT-OS/RIOT/blob/fc9577030e44b03aa279be21b4271f6b6d247c94/cpu/kinetis/ldscripts/kinetis.ld#L28